### ISSUE TITLE
Error in melo_server.dockerfile

### ISSUE THREAD

**Issue by Jaiaggarwaaaaal** [voxos-ai/bolna#323]
I am facing issue in melo_server.Dockerfile, the issue is 

> [melo-app  9/10] RUN pip install --no-cache-dir txtsplit torch torchaudio cached_path transformers mecab-python3==1.0.5 num2words==0.5.12 unidic_lite unidic pykakasi==2.2.1 fugashi==1.3.0 g2p_en==2.1.0 anyascii==0.3.2 jamo==0.4.1 gruut[de,es,fr]==2.2.3 g2pkk>=0.1.1 librosa==0.9.1 pydub==0.25.1 eng_to_ipa==0.0.2 inflect==7.0.0 unidecode==1.3.7 pypinyin==0.50.0 cn2an==0.5.22 jieba==0.42.1 langid==1.1.6 tqdm tensorboard==2.16.2 loguru==0.7.2 git+https://github.com/kvinwang/python-crfsuite:
0.518   Running command git clone --filter=blob:none --quiet https://github.com/kvinwang/python-crfsuite /tmp/pip-req-build-ryuiph02
2.586   Running command git submodule update --init --recursive -q
38.27 ERROR: Cannot install gruut[de,es,fr]==2.2.3 and python-crfsuite 0.9.6 (from git+https://github.com/kvinwang/python-crfsuite) because these package versions have conflicting dependencies.
38.27 ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
38.42 
38.42 [notice] A new release of pip is available: 23.0.1 -> 24.1.1
38.42 [notice] To update, run: pip install --upgrade pip
------
failed to solve: process "/bin/sh -c pip install --no-cache-dir txtsplit torch torchaudio cached_path transformers mecab-python3==1.0.5 num2words==0.5.12 unidic_lite unidic pykakasi==2.2.1 fugashi==1.3.0 g2p_en==2.1.0 anyascii==0.3.2 jamo==0.4.1 gruut[de,es,fr]==2.2.3 g2pkk>=0.1.1 librosa==0.9.1 pydub==0.25.1 eng_to_ipa==0.0.2 inflect==7.0.0 unidecode==1.3.7 pypinyin==0.50.0 cn2an==0.5.22 jieba==0.42.1 langid==1.1.6 tqdm tensorboard==2.16.2 loguru==0.7.2 git+https://github.com/kvinwang/python-crfsuite" did not complete successfully: exit code: 1

**Comment by prateeksachan** [comment]
hey @Jaiaggarwaaaaal , did you try deploying this in GPU-based instance? melo_server and whisper_server require GPU instances as mentioned in their docker config.

https://github.com/bolna-ai/bolna/blob/aafb2133ce2f1b9a837c57eb2b8e3efb7e643833/examples/whisper-melo-llama3/docker-compose.yml#L68-L82

**Comment by Jaiaggarwaaaaal** [comment]
No, @prateeksachan i didn't. @marmikcfc also tried to solve the issue in the Hackathon(AI - AGENT ), he also proposed that might be the GPU issue, because he was able to run 'bolna' in his device through instance.  I will try deploying it using the gpu.
